### PR TITLE
Fix ui_deployment on ODF4.11

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@oviner.tlv.csb>

### CODE REVIEW

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/ocs/ui/deployment_ui.py [hunk 1]
```diff
@@ -244,24 +246,18 @@ def install_internal_cluster(self):
...
-            logger.info("Configure Storage Class (thin on vmware, gp2 on aws)")
-            self.do_click(
-                locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
-            )
-            self.do_click(
-                locator=self.dep_loc[self.storage_class], enable_screenshot=True
-            )
+        logger.info("Configure Storage Class (thin on vmware, gp2 on aws)")
+        self.do_click(
+            locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
```
Comment (shyRozen) [inline ocs_ci/ocs/ui/deployment_ui.py:258]: Is the screenshot for debugging purpose? If yes please remove it

**Review by shyRozen — APPROVED** [review]

**Comment by shyRozen** [comment]
Please add `verfied` label

File: ocs_ci/ocs/ui/deployment_ui.py (hunk 1 above)
Comment (shyRozen) [reply ocs_ci/ocs/ui/deployment_ui.py:258]: Incase the test fails?

Comment (OdedViner) [reply ocs_ci/ocs/ui/deployment_ui.py:258]: No, for example we want to verify we have gp3 on AWS... so we can see all the options.. the default sc  in our deployment is gp2

**Comment by OdedViner** [comment]
> Please add `verfied` label

Done

**Review by petr-balogh — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**